runtime/onert/backend/cpu/ops/SpaceToBatchNDLayer.cc

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "SpaceToBatchNDLayer.h"
+
+#include "OperationUtils.h"
+
+#include <cker/operation/SpaceToBatchND.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace cpu
+{
+namespace ops
+{
+SpaceToBatchNDLayer::SpaceToBatchNDLayer()
+    : _input(nullptr), _block_shape(nullptr), _padding(nullptr), _output(nullptr)
+{
+  // DO NOTHING
+}
+
+template <typename T> void SpaceToBatchNDLayer::SpaceToBatchND(int32_t pad_value)
+{
```

Comment:
I understand the code below follows from Resizing the output tensor from TFLite. Except that in this case, it appears to be a validity check. Would it be possible to resize the output tensor here if the dimensions do not match?

Reply:
@dr-venkman No, resizing output tensor is not included here. This only checks dimension and throws error. 
We have dynamic shape inferer for dynamic tensor resizing. I am now working on implementation of shape inferer for this operator. After that, I will check if this dimension checking can be removed or can be changed into assertion.